packages/dataviews/src/components/dataviews-filters/search-widget.tsx

```diff
@@ -81,6 +75,30 @@ function generateFilterElementCompositeItemId(
 	return `${ prefix }-${ filterElementValue }`;
 }
 
+const Checkbox = ( { checked }: { checked: boolean } ) => {
+	return (
+		<span
+			className={ clsx(
+				'dataviews-filters__search-widget-listitem-checkbox',
+				{ 'is-checked': checked }
+			) }
+		>
+			{ checked && <Icon icon={ check } /> }
+		</span>
+	);
+};
+
+const Radio = ( { checked }: { checked: boolean } ) => {
+	return (
+		<span
+			className={ clsx(
+				'dataviews-filters__search-widget-listitem-radio',
+				{ 'is-checked': checked }
+			) }
+		/>
+	);
+};
+
```

Comment:
Calling these two components "checkbox" and "radio" implies the wrong abstraction and semantics. Checkbox and Radio semantics are different from listbox semantics. 

These should likely be called something like `SingleSelectionOption` and `MultiSelectionOption`, or something similar, and have a `selected` prop (instead of `checked`). Classnames should follow similar naming, too.

Reply:
Thanks for the feedback! I've updated the code to use the suggested naming and semantics in 76617fcac6e7b7310d9f6455690edcf5f0bdd5a3.